services/ordering/src/EShopOnAbp.OrderingService.Application.Contracts/Orders/IOrderAppService.cs

```diff
@@ -10,5 +11,10 @@ public interface IOrderAppService : IApplicationService
     Task<OrderDto> CreateAsync(OrderCreateDto input);
     Task<OrderDto> GetAsync(Guid id);
     Task<List<OrderDto>> GetMyOrdersAsync(GetMyOrdersInput input);
+    Task<List<OrderDto>> GetOrdersAsync(GetOrdersInput input);
     Task<OrderDto> GetByOrderNoAsync(int orderNo);
+    Task SetAsCancelledAsync(Guid id, SetAsCancelledDto input);
```

Comment:
Should only use id or orderNo parameter like **SetAsShipped**. No need for dto. See the review at dto.